boot/freeldr/freeldr/include/ver.h

```diff
@@ -19,21 +19,15 @@
 
 #pragma once
 
-/* Just some stuff */
 #define VERSION         "FreeLoader v3.2"
```

Comment:
Moving that one below (after the define FREELOADER_MAJOR/MINOR_VERSIONà, we could probably then do:
```c
#define VERSION         "FreeLoader v" TOSTRING(FREELOADER_MAJOR_VERSION) "." TOSTRING(FREELOADER_MINOR_VERSION)
```
which then looks like the `FrLdrVersionString` variable in the other file could be directly set to (i.e. `const PCSTR FrLdrVersionString = VERSION;` , if this compiles fine) :thinking:

Reply:
I will try that...

Reply:
see commit3 https://github.com/reactos/reactos/pull/7384/commits/e85e632ed2fc71e75886c1fb27542df29ce08a39 and commit4 https://github.com/reactos/reactos/pull/7384/commits/a156327dd0acb6b3b15e2b5bba70ca48cf93a6db